### DISCUSSION TITLE
Publish 오류 재문의 드립니다.

### DISCUSSION THREAD

**Discussion by PINK-J** [naverz/zepeto-studio-kor#1046] [5. Q&A]
안녕하세요! 어제 Publish 오류로 문의를 드렸었는데, 멀티서버 문제인 것 같다고 답변을 받았습니다..!

제가 지금 다른 분들과 같이 배우면서 월드를 만들고 있어서 멀티서버가 포함된 파일을 똑같이 제공 받았습니다..

다른 분들은 같은 파일로 정상적으로 작동하고 있는데,, 그래도 멀티서버 문제일 수 있을까요?

*오늘 프로젝트를 새로 만들어서 멀티서버가 포함된 파일을 다시 임포트하고, 바로 Publish 을 해봤을때 오류 입니다!
![unknown](https://user-images.githubusercontent.com/109596692/179998877-3c0533ae-e85d-4152-b7bf-492d7152bede.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 다른 분이 만든 파일을 받았다면 메타파일 문제일 수도 있겠다는 생각이 듭니다.🤔
멀티플레이 공부할 겸 처음부터 구현해 보는 것을 가장 추천 드리며
https://studio.zepeto.me/guides/multiplay-tutorial
멀티서버가 구현된 것을 원하신다면 멀티플레이 샘플을 깃에서 받아서 제작하시는 것도 좋은 방법입니다. 
https://github.com/naverz/zepeto-multiplay-example
